Source/Core/DolphinQt/Config/Mapping/HotkeyControllerProfile.h

```diff
@@ -22,5 +22,5 @@ class HotkeyControllerProfile final : public MappingWidget
   void CreateMainLayout();
 
   // Main
-  QHBoxLayout* m_main_layout;
+  QGridLayout* m_main_layout;
```

Comment:
It looks like this doesn't need to be a member variable as it's only used in `CreateMainLayout`.
You can just declare a local variable there and completely kill the above forward-declaration and include.
I realize you didn't add this member but now is a good time to fix it. :P

Reply:
OK, will change it to a local variable.